app/resources/coins/orangelight_coin_builder.rb

```diff
@@ -17,11 +17,18 @@ def clean_document(hash)
       hash.delete_if do |_k, v|
         v.nil? || v.try(:empty?)
       end
+    rescue NoMethodError
```

Comment:
I wonder if rescuing an exception here, it would be better from the user's point of view to return an error hash like we do for GeoBlacklight [documents](https://github.com/pulibrary/figgy/blob/master/app/services/geo_discovery/geoblacklight_document.rb#L32-L35). Something like...

```
def build
  if parent
    clean_document(document_hash)
  else
    {error: "#{decorator.title.first} with id: #{decorator.id} has no parent numismatic issue and cannot build an OL document."}
  end
end
```

Reply:
I think this is only in figgy and I'm not sure if the user would visit ../orangelight to check the OL document. This was my initial thought to rescue it and log it for our eyes and not the user's.

Reply:
but I'm happy to check it. Thanks for reviewing!

Reply:
Updated with the error message in clean_document(hash).

Reply:
I'm concerned that rescuing might mask other errors, and I would consider you and I to be "users" when we manually visit `../orangelight`. For me, getting the feedback directly is way easier than digging through the logs.

Reply:
yes, I did update maybe you missed the comment above :)